diff --git a/sample_wcl.py b/sample_wcl.py
index 70f8c6c..bd645c0 100644
--- a/sample_wcl.py
+++ b/sample_wcl.py
@@ -23,7 +23,7 @@ class SampleWCL:
         """
 
         # 指定階数で計測したデータを抽出
-        self.rssi = self.rssi[str(floor)]
+        rssi_floor_data = self.rssi[str(floor)]
 
         """
         以下、授業資料より抜粋
@@ -31,10 +31,10 @@ class SampleWCL:
         """
         # 1. 指定階数のAPを抽出
         check_str = str(floor) + "F"
-        rssi_floor_only = self.rssi[self.rssi["AP_name"].str.contains(check_str)]
+        rssi_floor_only = rssi_floor_data[rssi_floor_data["AP_name"].str.contains(check_str)]
         print(f"{floor}階のAP\n{rssi_floor_only}\n")  # デバッグ用
 
-        # 2. 位置PのRSSIデータを抽出
+        # 2. 位置PのRSSIデータを抽出（mainでまとめてできるように変更）
         rssi_p = rssi_floor_only[rssi_floor_only["Location index P"] == p]
         print(f"位置PのRSSIデータ\n{rssi_p}\n")  # デバッグ用
 
